protos/feast/core/FeatureView.proto

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2020 The Feast Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+syntax = "proto3";
+package feast.core;
+
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+option java_outer_classname = "FeatureViewProto";
+option java_package = "feast.proto.core";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "feast/core/DataSource.proto";
+import "feast/core/Feature.proto";
+
+message FeatureView {
+    // User-specified specifications of this feature table.
+    FeatureViewSpec spec = 1;
+
+    // System-populated metadata for this feature table.
+    FeatureViewMeta meta = 2;
+    string project = 3;
```

Comment:
One thing that stands out here. Why is the project not defined as part of the spec?

Reply:
I think we should move the `project` into the spec itself. I dont think its a breaking change since the `project` id was only recently introduced on master. @oavdeev what do you think? (it can be a separate PR).

Reply:
yeah I just copied FeatureTable, not opposed to moving it, but we'd have to move it in both places then

Reply:
Ok, follow up it is then.